worlds/pokemon_emerald/options.py

```diff
@@ -540,11 +762,53 @@ class ReceiveItemMessages(Choice):
     option_none = 2
 
 
+class RemoteItems(Toggle):
+    """
+    Instead of placing your own items directly into the ROM, all items are received from the server, including items you find for yourself.
+
+    This enables co-op of a single slot and recovering more items after a lost save file (if you're so unlucky).
+    But it changes pickup behavior slightly and obviously requires connection to the server to receive any items.
```

Comment:
I'd recommend removing the word "obviously" here for tone

Reply:
Changed in 46f7c86